requirements.txt

```diff
@@ -6,3 +6,4 @@ biopandas==0.5.1.dev0
 scipy>=1.8.0
 matplotlib>=3.5.1
 tqdm>=4.57.0
```

Comment:
Maybe we should add a note on `tqdm`, it is only a dependency because we use it in some of our sample scripts but it is not really a dependency of pyMBE as a module

Reply:
A comment line was added.